setup/tinyHIPPOstart

```diff
@@ -28,8 +28,9 @@ uci set luci.ccache.enable=0
 uci commit luci
 /etc/init.d/uhttpd restart
 
-# Create the sqlite database
+# Create and populate the sqlite database
 sqlite3 $TINYHIPPO_INSTALL_PATH/tinyHIPPO.db < $TINYHIPPO_INSTALL_PATH/tinyHIPPO_db_setup.sql
+sqlite3 $TINYHIPPO_INSTALL_PATH/tinyHIPPO.db < $TINYHIPPO_INSTALL_PATH/tinyHIPPO_populate_equations.sql
```

Comment:
When you merge the logic from the other file into this one you can remove this line from the tinyHIPPOstart script